ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoSpec.scala

```diff
@@ -1103,6 +1103,57 @@ class JdbcLedgerDaoSpec
         res3 shouldBe a[LedgerEntry.Rejection]
       }
     }
+
+    "be able to use divulged contract in later transaction" in {
+      val let = Instant.now
+      val offset1 = nextOffset()
+      val offset2 = nextOffset()
+      val offset3 = nextOffset()
+      def emptyTxWithDivulgedContracts(id: Long) = PersistenceEntry.Transaction(
```

Comment:
To improve readability of this test case please move this into a helper function.

Reply:
it is a helper function?

Reply:
Sorry, I wasn't specific enough: it should be moved outside of your test case (it's just details that you don't need at the first pass).